playbooks/common/openshift-loadbalancer/service.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Populate g_service_nodes host group if needed
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tasks:
+  - fail: msg="new_cluster_state is required to be injected in this playbook"
+    when: new_cluster_state is not defined
```

Comment:
So, I think the idea was to have playbooks to individually control service state for parts of the cluster for coordinating outages, etc.

Since we put that logic in our upgrade playbooks, I would argue that having separate playbooks for managing the overall "service" of a component is not needed.